dev-vcs/git-cola/git-cola-4.2.1-r1.ebuild

```diff
@@ -38,6 +38,7 @@ BDEPEND="sys-devel/gettext
 		)
 	")
 "
+PATCHES=( "${FILESDIR}/pyproject.toml.patch" )
```

Comment:
Thats not a great patch name.

Convention is ${P}-<what it does>.patch

So something like:
git-cola-4.2.1-fix-setuptools.patch

Reply:
Also include links to the bug and commit in git-cola to the patch header.